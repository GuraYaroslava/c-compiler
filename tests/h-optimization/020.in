int main()
{
    if (0)
    {
        return 0;
        printf("bodyIf");
    }
    else
    {
        return 0;
        printf("bodyElse");
        printf("bodyElse");
        printf("bodyElse");
        printf("bodyElse");
    }
    return 0;
}
