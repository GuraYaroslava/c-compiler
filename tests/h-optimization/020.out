table 0:
`int`
`float`
`char`

table 1:
variable `main` type of function() returning `int`
{
{
return
    0
}
return
    0
}


